### PR TITLE
Feature/precise tagger

### PR DESCRIPTION
**Pull request by chrisveilleux**
## Description
Re-implement the Precise tagger in Angular

## How to test
UI is running on precise.mycroft-test.net

### CODE REVIEW

File: projects/precise/src/environments/environment.ts [hunk 1]
```diff
@@ -28,7 +28,7 @@ export const environment = {
         forum: 'https://community.mycroft.ai',
         marketplace: 'http://market.mycroft.test',
         mimic: 'http://mimic.mycroft.ai',
```
Comment (krisgesling) [inline projects/precise/src/environments/environment.ts]: There seems to be a handful still using http and some using the `mycroft.test` domain

**Review by krisgesling — COMMENTED** [review]
Hey it seems that most of the api endpoints on the test server are throwing 502 errors, are these intentionally not active or is something borked?

Few other little things, and in case you didn't see it I made some comments in the JIRA ticket yesterday. 

I also noticed that the "Wake Word" nav menu item opens in a new tab which I assume is just inherited from the other items in that section. Small thing but worth changing I think.

File: projects/precise/src/app/modules/tagger/pages/tag/tag.component.ts [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    animations: [
+        trigger('newQuestion', [
+            state('untagged', style({opacity: 1})),
+            state('tagged', style({opacity: 0})),
+            transition(':enter', [
+                style({opacity: 0}),
+                animate('0.5s', style({opacity: 1}))
+            ]),
+            transition('untagged => tagged', animate('0.5s')),
+            transition('tagged => untagged', animate('0.5s'))
```
[the remark this answers is not in the document]
Comment (chrisveilleux) [reply projects/precise/src/app/modules/tagger/pages/tag/tag.component.ts:26]: The API call takes a second to load so I don't know if speeding these up will buy anything.

File: projects/precise/src/environments/environment.ts (hunk 1 above)
Comment (chrisveilleux) [reply projects/precise/src/environments/environment.ts]: This has been resolved.

File: projects/account/src/app/shared/models/wake-word.model.ts [hunk 3]
```diff
@@ -18,6 +18,6 @@ and limitations under the License.
 
 export interface WakeWord {
     id: string;
-    displayName: string;
-    settingName: string;
+    engine: string;
```
[the remark this answers is not in the document]
Comment (chrisveilleux) [reply projects/account/src/app/shared/models/wake-word.model.ts:21]: Not right now, but there is a plan to show engine when we get to the point where we have multiple wake words using precise

File: projects/precise/src/environments/environment.ts (hunk 1 above)
Comment (chrisveilleux) [reply projects/precise/src/environments/environment.ts]: Fixed

**Review by krisgesling — APPROVED** [review]

**Pull request merged**